app/src/main/java/fr/free/nrw/commons/category/CategoriesModel.kt

```diff
@@ -149,6 +150,13 @@ class CategoriesModel @Inject constructor(
         return categoryClient.searchCategories(title, SEARCH_CATS_LIMIT).toObservable()
     }
 
+    /**
+     * check if category exists
+     */
+    fun checkIfCategoryExists(query: String): Single<List<CategoryItem>> {
```

Comment:
Could you please rename to `checkWhetherCategoryExists`?
The name `checkIfCategoryExists` might be misunderstood as "_if category exists, then check_".